app/controllers/ems_common.rb

```diff
@@ -848,7 +850,7 @@ def set_record_vars(ems, mode = nil)
       creds[:ssh_keypair] = {:userid => @edit[:new][:ssh_keypair_userid], :auth_key => @edit[:new][:ssh_keypair_password]}
     end
     if ems.supports_authentication?(:bearer) && !@edit[:new][:bearer_token].blank?
-      creds[:bearer] = {:auth_key => @edit[:new][:bearer_token], :userid => "_"} # Must have userid
+      creds[:bearer] = {:auth_key => @edit[:new][:bearer_token], :userid => @edit[:new][:bearer_userid] || "_"}
```

Comment:
@oschreib I don't think we want to allow an empty `:bearer_userid`. I would make it mandatory (ui too).